app/views/activities/index.html.haml

```diff
@@ -0,0 +1,25 @@
+.container
+  .row
+    .col-md-8
+      %h3.align-center.blue View Previous Logs
+      %form.form-inline
+        #date-range.input-group
+          %input.form-control.date-picker{:type => "text", :name => "start"} 
```

Comment:
Oh good
So despite now screaming headache things had sunk in 
T

Sent from my iPad

> On Jun 16, 2015, at 7:08 AM, Levi Cole <notifications@github.com> wrote:
> 
> In app/views/activities/index.html.haml:
> 
> > @@ -0,0 +1,25 @@
> > +.container
> > +  .row
> > +    .col-md-8
> > +      %h3.align-center.blue View Previous Logs
> > +      %form.form-inline
> > +        #date-range.input-group
> > +          %input.form-control.date-picker{:type => "text", :name => "start"} 
> Yes, this is where the data is coming from. So, we need to change the name attribute to "start_date".
> 
> —
> Reply to this email directly or view it on GitHub.
>